### PR TITLE
Improved ascii/unicode masterpiece

### PR DESCRIPTION
**Pull request by clstrfsck**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.2
+// swift-tools-version:5.1
```
Comment (clstrfsck) [inline Package.swift:1]: Downgrade for 10.14

File: Sources/TicTacToe/TicTacToe/Board/TicTacToe+Board+ASCII.swift [hunk 2]
```diff
@@ -10,16 +10,25 @@ import Foundation
 // MARK: Board ASCII
 public extension TicTacToe.Board {
     func ascii() -> String {
-        let rowSeparator: String = .init(repeating: "-", count: 13) + "\n"
-        var output = rowSeparator
-        for row in Self.rows {
-            defer { output += "\n" + rowSeparator }
-            output += row.map({ "| \(ascii(square: $0)) "}).joined() + "|"
-        }
+        let rowSeparator: String = "\n───┼───┼───\n"
```
Comment (clstrfsck) [inline Sources/TicTacToe/TicTacToe/Board/TicTacToe+Board+ASCII.swift]: Using unicode box chars, and drawing without outside borders

File: Sources/TicTacToe/TicTacToe/Board/TicTacToe+Board+Fill.swift [hunk 3]
```diff
@@ -9,7 +9,7 @@ import Foundation
 
 public extension TicTacToe.Board {
     enum Fill: String, Equatable {
-        case cross = "✖"
-        case nought = "◯"
+        case cross = "\u{1b}[31;1mx\u{1b}[m"
+        case nought = "\u{1b}[32;1mo\u{1b}[m"
```
Comment (clstrfsck) [inline Sources/TicTacToe/TicTacToe/Board/TicTacToe+Board+Fill.swift:13]: Colours!

**Comment by Sajjon** [comment]
Nice! :D I took the liberty to further improve the ASCII logic (I did not even know I could push commits to your fork, but hey it worked ([this Github tutorial was helpful](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork))).

Take a look at my commit :), getting really neat now, thanks to your improvements and this last one :D

**Pull request merged**